iothub/service/src/Amqp/AmqpCbsSessionHandler.cs

```diff
@@ -20,7 +20,6 @@ internal class AmqpCbsSessionHandler : IDisposable
         private readonly IotHubConnectionProperties _credential;
 
         private AmqpCbsLink _cbsLink;
-        private readonly EventHandler _connectionLossHandler;
```

Comment:
Unused. CBS links don't allow for subscribing to link drop events for some reason.